torch/nn/quantizable/modules/activation.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, embed_dim: int, num_heads: int,
         self.out_proj = nn.Linear(self.embed_dim, self.embed_dim, bias=bias, **factory_kwargs)  # type: ignore[assignment]
 
         # Functionals
-        self.q_scaling_product = nnq.FloatFunctional()
```

Comment:
what's the reasoning for this change?

Reply:
NOPE. Without this change there's a circular import that gets created by the PR. Not sure if there's a better way to address this though...